beta-src/src/components/ui/WDNotification.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import { Box } from "@mui/material";
+import GameNotification from "../../state/interfaces/GameNotification";
+
+interface WDNotificationProps {
+  notification: GameNotification;
+}
+
+const WDNotification: React.FC<WDNotificationProps> = function ({
+  notification,
+}): React.ReactElement {
+  return (
+    <Box>
```

Comment:
Should this be using Box as well or can we use an empty frag  `</>` here? WDNotification is already being wrapped by a Box on WDDeletePanel.tsx

Reply:
Empty frag and removing the box introduces errors, so i removed the boxes in WDDeletePanel.tsx, which works